ci/lint/04_install.sh

```diff
@@ -9,6 +9,7 @@ export LC_ALL=C
 travis_retry pip3 install codespell==1.15.0
 travis_retry pip3 install flake8==3.7.8
 travis_retry pip3 install yq
+travis_retry pip3 install mypy==0.700
```

Comment:
why did you set 0.700 instead of 0.770 like you mentioned in [this comment](https://github.com/bitcoin/bitcoin/pull/18210#discussion_r433256265)?

Reply:
I screwed up.

Reply:
I think this should be bumped because version 700 can not be compiled with python3.8-dev